server/jetstream.go

```diff
@@ -1645,6 +1647,8 @@ func (js *jetStream) usageStats() *JetStreamStats {
 	var stats JetStreamStats
 	js.mu.RLock()
 	stats.Accounts = len(js.accounts)
+	stats.ReservedMemory = (uint64)(js.memReserved)
+	stats.ReservedStore = (uint64)(js.storeReserved)
```

Comment:
these are zero when dynamic limits are used rather than fixed ones from config, would be nice if they adjust to the dynamically reserved level rather than return 0